src/Apps/Artist/Routes/AuctionResults/__tests__/AuctionResults.test.tsx

```diff
@@ -267,6 +268,30 @@ describe("AuctionResults", () => {
             })
           })
         })
+        describe("year created filter", () => {
```

Comment:
This just adds the missing year created filter test.

Paired w/ @lilyfromseattle on trying to get one created for the reset link, but we'll have to revisit that.